Install/install.sh

```diff
@@ -99,10 +99,12 @@ sudo python setup.py install
 sudo python3 setup.py install
 
 # module for interfacing with the keyboard
-sudo pip install curtsies
 sudo pip3 install curtsies
-sudo pip install numpy
 sudo pip3 install numpy
+sudo pip3 install python-periphery
```

Comment:
@johnisanerd Robert and I agree that we should remove the periphery installation from the GPG3 repo. It is used by GPG (for the line follower), and DI_Sensors, but not directly by anything in GPG3. Applies to this line (104) as well as line 107.